flink-cdc-connect/flink-cdc-source-connectors/flink-connector-db2-cdc/src/main/java/com/ververica/cdc/connectors/db2/source/utils/Db2TypeUtils.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.db2.source.utils;
+
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.types.DataType;
+
+import io.debezium.relational.Column;
+
+import java.sql.Types;
+
+/** Utilities for converting from Db2 types to Flink types. */
+public class Db2TypeUtils {
+
+    /** Returns a corresponding Flink data type from a debezium {@link Column}. */
+    public static DataType fromDbzColumn(Column column) {
+        DataType dataType = convertFromColumn(column);
+        if (column.isOptional()) {
+            return dataType;
+        } else {
+            return dataType.notNull();
+        }
+    }
+
+    /**
+     * Returns a corresponding Flink data type from a debezium {@link Column} with nullable always
+     * be true.
+     */
+    private static DataType convertFromColumn(Column column) {
+        switch (column.jdbcType()) {
+            case Types.CHAR:
+            case Types.VARCHAR:
+            case Types.SQLXML:
+                return DataTypes.STRING();
+            case Types.CLOB:
+            case Types.BLOB:
+            case Types.BINARY:
+            case Types.VARBINARY:
+                return DataTypes.BYTES();
+            case Types.INTEGER:
+            case Types.SMALLINT:
+                return DataTypes.INT();
+            case Types.BIGINT:
+                return DataTypes.BIGINT();
+            case Types.FLOAT:
+            case Types.REAL:
+            case Types.DOUBLE:
+            case Types.DECIMAL:
+            case Types.NUMERIC:
+                return DataTypes.DECIMAL(column.length(), column.scale().orElse(0));
```

Comment:
What is this for converting FLOAT/DOUBLE to DataTypes.DECIMAL? There is no special handling in [Db2ValueConverters](https://github.com/debezium/debezium-connector-db2/blob/86b3f156f5e7544c5f746387e4749025b5685e0e/src/main/java/io/debezium/connector/db2/Db2ValueConverters.java#L61).

Reply:
I modify it. Thank you.